storages/python/tests/test_blob_storage.c

```diff
@@ -0,0 +1,50 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "minunit/minunit.h"
+
+#include "utils/integers.h"
+#include "dlite.h"
+#include "dlite-macros.h"
+#include "dlite-storage-plugins.h"
+
+
+DLiteInstance *inst = NULL;
+
+
+MU_TEST(test_load)
+{
+  char *url = "blob://" STRINGIFY(CURRENT_SOURCE_DIR) "/test_blob_storage.c";
+  inst = dlite_instance_load_url(url);
+}
```

Comment:
It would be natural to run mu_check(inst) here, since test_load tests loading of the blob. If the loading was unsuccessful, that failure should fail the test_load test itself, not just the later attempt to save the blob.

Reply:
Good point. Done